.github/workflows/wheels.yml

```diff
@@ -26,7 +26,9 @@ jobs:
           submodules: 'true'
 
       - name: Build wheels
-        uses: pypa/cibuildwheel@v2.11.1
+        uses: pypa/cibuildwheel@v2.11.2
+        env:
+            CIBW_SKIP: cp36-* cp37-* pp37-*
```

Comment:
Setting this ignores the skipping in https://github.com/inducer/pyopencl/blob/main/pyproject.toml#L34 which results in cibw building `cp38-win32` builds (windows i686 builds)

Reply:
Thanks! In retrospect, I could have figured this out. Thanks for the help!

Reply:
:smile: It took me an embarrassing amount of time and a tmate session to figure out that we were building for i686.

Reply:
You can tmate into Windows CI runners?!

Reply:
Yes. It drops you into a msys2 shell.

Reply:
That's super useful to know, and it makes me less averse to (at least trying to) support Windows.